src/detailed/introduction.md

```diff
@@ -0,0 +1,45 @@
+# Eludris Detailed
+Dedicated and detailed rundown of how we *plan* Eludris to eventually function.
+This is mostly sketched out from and is and extension of the old `plan.md` in our `eludris/eludris` repository.
+
+## Overview
+
+The goal is to provide a uniquely fresh but not totally new experience focused around any topic or interest whilst having a policy of being secure, free (as in monetary value), free (as in freedom), privacy respecting, open source, and decentralized without putting users who may not have knowledge or anything to do with the aforementioned topics at an inconvenience.
+
+In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris is more than welcome.
```

Comment:
Extremely nitty, but I'd use `--` (or an actual em-dash) for the interjection just to reduce the number of commas in this section. I think that should make it slightly easier to read.